lms/djangoapps/course_api/tests/test_views.py

```diff
@@ -85,8 +85,13 @@ def test_as_inactive_user(self):
         self.verify_response(params={'username': inactive_user.username})
 
     def test_missing_username(self):
+        response_to_anonymous = self.verify_response(expected_status_code=200)
```

Comment:
In order to really test that missing_username is equivalent to anonymous, you would need to setup the test so that there's a course that anonymous users wouldn't have access to.  I'm not sure what exactly that implies.  But without such a distinction, this test is equivalently a no-op, right?